### PR TITLE
chore: change xdefi redirect link to chrome store

### PR DESCRIPTION
**Pull request by loxator**
### Description
chore: change xdefi redirect link to chrome store

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [ ] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] This PR passes the Circle CI checks (if **merging a fork** run `yarn` with node version 12.22.7 to ensure no errors)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@ dist/
 package-lock.json
 .rpt2_cache
 .vscode
-*.patch
\ No newline at end of file
+*.patch
+.history/
```
Comment (Adamj1232) [inline .gitignore:8]: Can you please add an empty line at the bottom of this file if you have a moment?

**Review by Adamj1232 — CHANGES_REQUESTED** [review]
Thanks for this PR! Just a couple small changes

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bnc-onboard",
-  "version": "1.38.1",
+  "version": "1.38.2",
```
[the remark this answers is not in the document]
Comment (loxator) [reply package.json]: @Adamj1232 , thanks for pointing it out. I have addressed your comments :smile: .

**Review by Adamj1232 — APPROVED** [review]
Thank you!!

**Pull request merged**